### PR TITLE
codehawk rules v1.0

### PR DESCRIPTION
**Pull request by Jcseeco**

### CODE REVIEW

File: codehawk.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+description=Analyze Java code smell
+homepageUrl=https://github.com/SDPMLab/CodeHawk
+archivedVersions=1.0
+publicVersions=1.0
+
+defaults.mavenGroupId= org.codehawk.plugin
+defaults.mavenArtifactId= codehawk
+
+1.0.description= Initial version compatible with SonarQube 7.7
+1.0.sqVersions=[7.7,LATEST]
```
Comment (ganncamp) [inline codehawk.properties]: This should start at 7.9, the oldest version listed in the Marketplace

Comment (Jcseeco) [reply codehawk.properties]: Done, but our plugin still need fixing.

**Comment by Jcseeco** [comment]
I've updated the file, thank you!

File: codehawk.properties [hunk 2]
```diff
@@ -1,26 +1,14 @@
 category=External Analysers
 description=Analyze Java code smell
 homepageUrl=https://github.com/SDPMLab/CodeHawk
-archivedVersions=1.0
-publicVersions=1.0
+archivedVersions=1.2
```
Comment (ganncamp) [inline codehawk.properties]: It can't be both public and archived. You'll (probably) _move_ 1.2 to this list once you release 1.3

Comment (Jcseeco) [reply codehawk.properties]: Removed value of archivedVersions

**Pull request merged**